opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -592,7 +592,7 @@ def format_trace_id(trace_id: int) -> str:
         trace_id: Trace ID int
 
     Returns:
-        The trace ID as 32-byte hexadecimal string
+        The trace ID cast to 32-character hexadecimal string (16 bytes)
```

Comment:
```suggestion
        The trace ID as 32-bytes hexadecimal string (16 bytes)
```

Would just adding the size work for you? byte is more precise than character.

Reply:
These are different things, no? one character hexstring is 1/2 a byte. That's where the confusion stems from

Reply:
Correct